experiments/standalone/Snow/snowmip_simulation.jl

```diff
@@ -133,6 +186,13 @@ scatter!(
     T_snow .+ 273.15,
     label = "Data",
 )
+# # Plot the observed atmospheric temperature
```

Comment:
Did you want to include this?

we might also want to include the annual average plot for surface temperature.